ui/apps/platform/src/Containers/Violations/ViolationsTablePanel.tsx

```diff
@@ -33,14 +33,32 @@ import useToasts from 'hooks/patternfly/useToasts';
 import { resolveAlert } from 'services/AlertsService';
 import { excludeDeployments } from 'services/PoliciesService';
 import type { ListAlert } from 'types/alert.proto';
-import type { TableColumn } from 'types/table';
 import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
 import type { SearchFilter } from 'types/search';
 import type { OnSearchCallback } from 'Components/CompoundSearchFilter/types';
 import ResolveConfirmation from './Modals/ResolveConfirmation';
 import ExcludeConfirmation from './Modals/ExcludeConfirmation';
 import ViolationsTableSearchFilter from './ViolationsTableSearchFilter';
 
+type TableColumn = {
```

Comment:
Instead of generalizing these types with generics, inline them in the only file that uses them and use the specific type of data expected in this single case.